shared/components/SpeakerSocialLinks/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+
+const IconLink = ({ present, url, type, title }) => {
+  if (!present) { return null; }
+  return (
+    <a
```

Comment:
That's not true. You can do all those things as children is handled by React, and all props are passed through.